all/generate/.github/workflows/cron.yaml

```diff
@@ -0,0 +1,52 @@
+name: Daily build and test (scheduled job)
```

Comment:
probably don't need the "scheduled job" bit here

Reply:
Agreed. I will remove.